server/index.ts

```
@@ -0,0 +1,736 @@
+            
+            const nodeName = xmlElement.nodeName ?? 'div';
+            const attrs = xmlElement.getAttributes ? xmlElement.getAttributes() : {};
+            
+            console.log(`🔍 YXmlElement: ${nodeName}`, attrs);
+            
+            // Get the text content from this element
+            let innerContent = '';
+            if (xmlElement.length > 0) {
+              for (let j = 0; j < xmlElement.length; j++) {
+                const grandChild = xmlElement.get(j);
+                if (grandChild && typeof grandChild === 'object' && (grandChild as any).constructor?.name === 'YXmlText') {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)